lib/auth/webauthncli/fido2_test.go

```diff
@@ -1646,6 +1735,11 @@ type fakeFIDO2Device struct {
 	// Causes libfido2.ErrNotFIDO2 on Info.
 	u2fOnly bool
 
+	// assertionErrors is a chain of errors to return from Assertion.
+	// Errors are returned from start to end and removed from the slice.
```

Comment:
```suggestion
	// Errors are returned from start to end and removed 1by1 on each execution of Assertion.
```
maybe it will be more readable that way?

Reply:
Done, good call.